paprika_to_md.py

```diff
@@ -62,7 +62,7 @@ def json_manipulate(sourceFolder, targetFolder, resourceFolder, template):
         for filename in files:
             if re.search(r'\.json$', filename):
                 p = Path(sourceFolder + filename)
-                with open(p) as json_file:
+                with open(p, encoding="utf-8") as json_file:
                     d = {}
                     recipe_data = json.load(json_file)
```

Comment:
Hi @jt196, 

Thanks for reviewing and merging!

No updates should be needed to the docs. Based on the [Paprika User Guide (For Windows)](https://www.paprikaapp.com/help/windows/#:~:text=The%20YAML%20format%20is%20quite%20strict%20with%20regards%20to%20whitespace%20and%20indentation.%20Please%20also%20make%20sure%20your%20file%20is%20saved%20with%20a%20UTF%2D8%20encoding.), Paprika encodes their recipe files in [UTF-8](https://en.wikipedia.org/wiki/UTF-8). This pull request _explicitly_ defines 'utf-8' as the file encoding in order to prevent errors when the recipes are loaded loaded from json using `json.load()`.

Reply:
Ah that makes a lot of sense as to why my installation was working and the other chap's wasn't. Well done for checking it out and updating.